Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp

```diff
@@ -126,27 +116,31 @@ namespace AZ::SceneAPI::SceneData
                 ->Version(1);
 
             serializeContext->Class<PrefabGroup, DataTypes::IPrefabGroup>()
-                ->Version(2) // added createProceduralPrefab
```

Comment:
hmm, this seems to remove the option to not generate a proc prefab per scene asset... is that desired?

Reply:
The toggle just did double duty over the group itself. With and without this change, you could turn off default prefabs by removing the change.

Before this change, the toggle and removing the group both turned off default prefabs, but the toggle wasn't as functional as removing the group. So this simplifies and consolidates the workflow to just removing the group.